themes/theme-b2b/src/index.ts

```diff
@@ -126,6 +127,7 @@ const button = {
       borderRight: '1px solid',
       borderLeft: '1px solid',
       borderBottom: 'none',
+      marginBottom: '1px',
```

Comment:
This doesn't uses the theme 😐

Reply:
hm this should be just an hack to fix the problem that the Select jumps 1px if its open. Do you have a better solution? I know thats not part of our theme space values..

Reply:
Is it a hack or a fix? 🤪

Do you know what causes the jumping?

Reply:
yes because of the border usage. But i cant use outline because in outline you cant set different borders at left right bottom or top. You only can set the same for all four.

Reply:
Buuuut, you can make the border transparent instead of adding margins that also shift the layout! :)

Reply:
I hope this is a good solution 😬